### PR TITLE
Updates docs and env sample files to address the neglected SP

### LINKED ISSUE #268: [BUG] SP to Key Vault script fails when following quickstart guide
The script to insert service principal client ID and secret to Key Vault fails when following the quickstart guide and running locally due to the environment variables not being documented.

The environment variables required by the script `/devops/scripts/set_contributor_sp_secrets.sh` needs to be added to:
- the quickstart document
- the `.env` file (sample)

### PR DESCRIPTION
**Pull request by tompaana**
Fixes #268 

## How is this addressed

- Updates the documentation (`quickstart.md`) to explain the required environment variables
- Updates `.env` sample files to contain the required variables

### CODE REVIEW

File: devops/.env.sample [hunk 1]
```diff
@@ -1,11 +1,13 @@
-# Used by backend state 
-MGMT_STORAGE_ACCOUNT_NAME=__CHANGE_ME__
+ARM_TENANT_ID=__CHANGE_ME__
+ARM_SUBSCRIPTION_ID=__CHANGE_ME__
+ARM_CLIENT_ID=__CHANGE_ME__
+ARM_CLIENT_SECRET=__CHANGE_ME__
+SUB_ID=__CHANGE_ME__
```
Comment (Lybecker) [inline devops/.env.sample:5]: Would people know that SUB_ID is subscription ID?

File: devops/scripts/set_contributor_sp_secrets.sh [hunk 2]
```diff
@@ -9,14 +9,14 @@ set -e
 #
 # Required environment variables:
 #
-#   - TF_VAR_tre_id - The TRE ID, used to deduce the Key Vault name
-#   - SUB_ID - The Azure subscription ID
+#   - TRE_ID - The TRE ID, used to deduce the Key Vault name
+#   - ARM_SUBSCRIPTION_ID - The Azure subscription ID
```
Comment (Lybecker) [inline devops/scripts/set_contributor_sp_secrets.sh:13]: Here you are removing SUB_ID and adding ARM_SUBSCRIPTION_ID. In the .env.sample SUB_ID is used

**Review by Lybecker — APPROVED** [review]
LGTM - a few comments about SUB_ID vs SUBSCRIPTION_ID. It seems like there is a discrepancy

File: devops/.env.sample (hunk 1 above)
Comment (tompaana) [reply devops/.env.sample]: You caught a "bug" - that was not suppose to be there anymore. Thanks!

File: devops/scripts/set_contributor_sp_secrets.sh (hunk 2 above)
Comment (tompaana) [reply devops/scripts/set_contributor_sp_secrets.sh:13]: Sorry, it is fixed now.

**Review by Lybecker — APPROVED** [review]
LGTM

**Pull request merged**